packages/rmpc/build.sh

```diff
@@ -0,0 +1,25 @@
+TERMUX_PKG_HOMEPAGE="https://mierak.github.io/rmpc/"
+TERMUX_PKG_DESCRIPTION="A configurable TUI MPD client with album art support"
+TERMUX_PKG_LICENSE="BSD 3-Clause"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_SUGGESTS="cava, ffmpeg, python-yt-dlp"
+TERMUX_PKG_VERSION="0.10.0"
+TERMUX_PKG_SRCURL="https://github.com/mierak/rmpc/archive/v$TERMUX_PKG_VERSION.tar.gz"
+TERMUX_PKG_SHA256=86e6b465707cdca82f53aed2a3bef600430edbf4e626124d194ac346ee989cbd
+TERMUX_PKG_AUTO_UPDATE=true
+TERMUX_PKG_BUILD_IN_SRC=true
+
+termux_step_make() {
+	termux_setup_rust
+	cargo build --jobs $TERMUX_PKG_MAKE_PROCESSES --target $CARGO_TARGET_NAME --release --locked
+}
+
+termux_step_make_install() {
+	install -Dm755 -t $TERMUX_PREFIX/bin target/${CARGO_TARGET_NAME}/release/rmpc
+	install -Dm644 -t "$TERMUX_PREFIX/share/doc/$TERMUX_PKG_NAME/" README* CHANGELOG* FAQ*
```

Comment:
We don't usually add loose documentation files to packages.

The Arch package seems to go out of its way to add the `README.md`.
https://gitlab.archlinux.org/archlinux/packaging/packages/rmpc/-/blob/0.10.0-1/PKGBUILD?ref_type=tags#L37-56

Is there any particular rationale for shipping these?
Not not as easily accessible as manpages, so they'd only be useful if you know they're there.